tests/zsa.rs

```diff
@@ -231,6 +231,7 @@ struct TestOutputInfo {
 fn build_and_verify_bundle(
     spends: Vec<&TestSpendInfo>,
     outputs: Vec<TestOutputInfo>,
+    assets_to_burn: Vec<(AssetId, i64)>,
```

Comment:
just `burn: ....`

Reply:
Also, let's use `Vec<(AssetId, NoteValue)>` to be closer to the implementation.

Reply:
done